static/js/stream_data.js

```diff
@@ -46,6 +47,27 @@ exports.get_colors = function () {
     return _.pluck(exports.subscribed_subs(), 'color');
 };
 
+exports.get_subscribers_count = function (stream_name) {
+    var sub = exports.get_sub(stream_name);
+    if (typeof sub === 'undefined') {
+        return undefined;
+    }
+    var count = Object.keys(sub.subscribers._items).length;
+
+    if (sub.subscribed === false && sub.subscribers.has(page_params.email) === true) {
+        // If user is not subscribed (i.e. sub.subscribed is false)
```

Comment:
How can the user's email be in the subscribers list if `sub.subscribed` is false?